app/main.py

```diff
@@ -2,0 +1,38 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        self.add_alive()
+
+    def add_alive(self) -> None:
+        if self.health > 0 and self not in Animal.alive:
+            Animal.alive.append(self)
+        elif self.health <= 0 and self in Animal.alive:
+            Animal.alive.remove(self)
+
+    def __str__(self) -> str:
+        return (f"Name: {self.name}, "
+                f"Health: {self.health}, Hidden: {self.hidden}")
```

Comment:
Code Style: It's great that you're using string interpolation instead of concatenation. This makes the code more readable and efficient.